src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -66,22 +71,38 @@ export class HuiDialogEditCard extends LitElement {
       <hui-edit-card
         .hass="${this.hass}"
         .lovelace="${this._params.lovelace}"
-        .path="${this._params.path}"
         .cardConfig="${this._cardConfig}"
         .closeDialog="${this._cancel}"
+        .saveCard="${this._save}"
+        .newCard="${this._newCard}"
       >
       </hui-edit-card>
     `;
   }
 
   private _cardPicked(cardConf: LovelaceCardConfig): void {
     this._cardConfig = cardConf;
+    this._newCard = false;
```

Comment:
Why would it no longer be a new card if we picked a card? It would still need to be added instead of replaced?

Reply:
I changed the name of this variable and forgot to make it's meaning reflect it...